Dockerfile

```diff
@@ -1,7 +1 @@
-FROM quay.io/astronomer/astro-runtime:7.2.0
-
-# install python virtualenv to run dbt
-WORKDIR /usr/local/airflow
-COPY dbt-requirements.txt ./
-RUN python -m virtualenv dbt_venv && source dbt_venv/bin/activate && \
-    pip install --no-cache-dir -r dbt-requirements.txt && deactivate
```

Comment:
Since many versions of [Airflow and dbt conflict](https://github.com/astronomer/astronomer-cosmos/pull/779/files) (including Airflow 2.8), I suggest we keep the separate Python virtual environment for dbt dependencies - as it was before.

This will allow us to easily upgrade to `quay.io/astronomer/astro-runtime:10.0.0` (that uses Airflow 2.8) once Cosmos 1.2.6/1.3 is released (with the fix https://github.com/astronomer/astronomer-cosmos/pull/772).

Reply:
Fair point. I've gone back to the virtualenv setup. Also, because Cosmos 1.3.0 landed, I was able to bump the astro-runtime to 10.0.0 straightaway :-)